umm-spec-lib/resources/example-data/dif10/sample_collection.xml

```diff
@@ -0,0 +1,143 @@
+    </Spatial_Coverage>
+    <Project>
+      <Short_Name>short name</Short_Name>
+    </Project>
+    <Organization>
+      <Organization_Type>ARCHIVER</Organization_Type>
+      <Organization_Name>
+        <Short_Name>EU/JRC/IES</Short_Name>
+        <Long_Name>Institute for Environment and Sustainability, Joint Research Center, European Union</Long_Name>
+      </Organization_Name>
+      <Personnel>
+        <Role>DATA CENTER CONTACT</Role>
+        <Contact_Person>
+          <Last_Name>Not provided</Last_Name>
+        </Contact_Person>
+      </Personnel>
+    </Organization>
+    <Reference>
+      <Author>author</Author>
+      <Publication_Date>2015</Publication_Date>
+      <Title>title</Title>
+      <Series>1</Series>
+      <Edition>2</Edition>
+      <Volume>3</Volume>
+      <Issue>4</Issue>
+      <Report_Number>5</Report_Number>
+      <Publication_Place>Frederick, MD</Publication_Place>
+      <Publisher>publisher</Publisher>
+      <Pages>678</Pages>
+      <ISBN>978-0-394-80001-1</ISBN>
+      <Online_Resource>http://example.com</Online_Resource>
+      <Other_Reference_Details>blah</Other_Reference_Details>
+    </Reference>
+    <Summary>
+      <Abstract>summary of the dataset</Abstract>
+      <Purpose>A grand purpose</Purpose>
+    </Summary>
+    <Related_URL>
+      <URL>http://www.foo.com</URL>
+    </Related_URL>
+    <Metadata_Association>
+      <Entry_ID>
+        <Short_Name>COLLOTHER-237</Short_Name>
+        <Version>1</Version>
+      </Entry_ID>
+      <Type>Input</Type>
+      <Description>Extra data</Description>
+    </Metadata_Association>
+    <Metadata_Association>
+      <Entry_ID>
+        <Short_Name>COLLOTHER-238</Short_Name>
+        <Version>1</Version>
+      </Entry_ID>
+      <Type>Input</Type>
+      <Description>Extra data</Description>
+    </Metadata_Association>
+    <Metadata_Association>
+      <Entry_ID>
+        <Short_Name>COLLOTHER-239</Short_Name>
+        <Version>1</Version>
+      </Entry_ID>
+      <Type>Input</Type>
+      <Description>Extra data</Description>
+    </Metadata_Association>
+    <Metadata_Name>CEOS IDN DIF</Metadata_Name>
+    <Metadata_Version>VERSION 10.2</Metadata_Version>
+    <Metadata_Dates>
+      <Metadata_Creation>2000-03-24T22:20:41-05:00</Metadata_Creation>
+      <Metadata_Last_Revision>2000-03-24T22:20:41-05:00</Metadata_Last_Revision>
+      <Data_Creation>1970-01-01T00:00:00</Data_Creation>
+      <Data_Last_Revision>1970-01-01T00:00:00</Data_Last_Revision>
+    </Metadata_Dates>
+    <Additional_Attributes>
+      <Name>dif-additional-attribute</Name>
+      <DataType>FLOAT</DataType>
+      <Description>something float</Description>
+      <ParameterRangeBegin>0.1</ParameterRangeBegin>
+      <ParameterRangeEnd>100.43</ParameterRangeEnd>
+      <Value>12.3</Value>
+    </Additional_Attributes>
+    <Collection_Data_Type>SCIENCE_QUALITY</Collection_Data_Type>
+    <Extended_Metadata>
+    <Metadata>
+      <Group>gov.nasa.gsfc.gcmd</Group>
```

Comment:
For DIF10, the `Group` of Extended_Metadata will not be preserved after a round trip from DIF10 -> UMM-C -> DIF10 because there is no `Group` in DIF10 `AdditionalAttributes`. Is this something we should worry about?

Reply:
No - DIF 10 additional attributes go into UMM-C additional attributes, If a DIF 10 additional attribute gets translated into DIF 9 Extended_Metadata the additionalattribute group name will be added.  Also we are going to stop supporting dif 9 very soon.

Reply:
Also no, There shouldn't be any additional attributes in DIF 10 under Extended_Metadata.  But if there is we translate it to from UMM-C back to DIF 10 as AdditionalAttributes and not Extended_Metadata.

Reply:
DIF10 AdditionalAttributes parsing code gets AdditionalAttributes from both Extended_Metadata and AdditionalAttributes, and the Extended_Metadata parsing handles Group, but not the AdditionalAttributes parsing code. That's why I asked the question.